x/ethbridge/keeper/msg_server.go

```diff
@@ -64,6 +65,9 @@ func (srv msgServer) Lock(goCtx context.Context, msg *types.MsgLock) (*types.Msg
 	globalSequence := srv.Keeper.GetGlobalSequence(ctx, msg.NetworkDescriptor)
 	srv.Keeper.UpdateGlobalSequence(ctx, msg.NetworkDescriptor, uint64(ctx.BlockHeight()))
 
+	// we also take ibc token is bridge token, since it is not from sifnode
+	isBridgeToken := strings.HasPrefix(msg.DenomHash, "ibc/")
```

Comment:
What about if the token is rowan? This will work currently when we are on Ethereum, but what about when more chains are added?